src/isIsogram.test.js

```diff
@@ -6,4 +6,39 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+  
+  it('should return true for an empty string', () => {
+    const result = isIsogram('');
+
+    expect(result).toBe(true);
+  });
+
+  it('should return true for a lowercase isogram', () => {
+    const result = isIsogram('playgrounds');
+
+    expect(result).toBe(true);
+  });
+
+  it('should return false for a mixed-case word', () => {
+    const result = isIsogram('Adam');
+
+    expect(result).toBe(false);
+  });
+
+  it('should return false for a word with repeating letters', () => {
+    const result = isIsogram('look');
+
+    expect(result).toBe(false);
+  });
+
+  it('should handle accented characters', () => {
+    const result = isIsogram('résum');
+
+    expect(result).toBe(true);
+  });
+
+  it('should handle non-letter characters', () => {
+    const result = isIsogram('ab-cdefg');
```

Comment:
The test case 'should handle non-letter characters' may be misleading. If the function is supposed to ignore non-letter characters, then the test case is valid. However, if the function is supposed to consider non-letter characters as part of the isogram check, this test case could fail and should be revised.